plugins/badges/src/components/BadgesDrawer.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import { Button, Drawer, Grid, Typography } from '@material-ui/core';
+import { CodeSnippet, Content, InfoCard, useApi } from '@backstage/core';
+
+const BadgeCard = ({ title, badgeUrl, markdownCode }) => (
+  <Grid item>
```

Comment:
Should normally have the item in the same component as the parent container, suggest moving it down to `BadgesDrawerContent` to not spread out the concerns

Reply:
Dropped the drawer.